src/swap/components/SwapSettings.tsx

```diff
@@ -0,0 +1,37 @@
+import { isValidElement, useMemo } from 'react';
+import { SwapSettingsSvg } from '../../internal/svg/swapSettings';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsReact } from '../types';
+
+export function SwapSettings({
+  className,
+  icon,
+  title = 'Auto',
+}: SwapSettingsReact) {
+  const iconSvg = useMemo(() => {
+    if (icon === undefined) {
+      return SwapSettingsSvg;
+    }
+    if (isValidElement(icon)) {
+      return icon;
+    }
+  }, [icon]);
+
+  return (
+    <div
+      className={cn('flex w-full items-center justify-between', className)}
+      data-testid="ockSwapSettings_Settings"
+    >
+      <div className="flex items-center space-x-1">
+        <h4 className="font-medium text-base">{title}</h4>
```

Comment:
I don't think this should be an H4. As this is more a `inline-block` element, and not a `block` element.
This should probably be `span`.

Reply:
Updated to span